index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Acme Quiz</title>
+  <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+    <div class="container" >
+      <header><h1>Joe Dirt's Firework Stand</h1></header>
+      <div class="container">
+        <div class="navBar">
+          <div class="dropdown">
+            <button class="btn btn-default dropdown-toggle" type="button" id="dd" data-toggle="dropdown">Categories <span class="caret"></span></button>
+              <ul class="dropdown-menu" aria-labelledby="dd">
+                <li><a href="#" id="fire">Pretty Fireworks</a></li>
+                <li><a href="#" id="expl">Explosives</a></li>
+                <li><a href="#" id="kids">For Kids</a></li>
+              </ul>
+          </div>
+        </div>
+      </div>
+
+        <div class="container-fluid grid" id="fwGrid">
+        <!-- contents will display here -->
+
+        </div>
+
+    </div>
+  <script src="bower_components/jquery/dist/jquery.min.js" type="text/javascript"></script>
+  <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+  <script src="../js/load.js" type="text/javascript"></script>
```

Comment:
When your js directory is in the same directory as your index file (in this case the root), you do not need to `../` as that will tell the server to look back a directory. You can eliminate the `../` in both of these script files.